src/components/InfiniteHits/__tests__/__snapshots__/InfiniteHits-test.js.snap

```diff
@@ -72,6 +72,96 @@ exports[`InfiniteHits markup should render <InfiniteHits /> on last page 1`] = `
 </div>
 `;
 
+exports[`InfiniteHits markup should render <InfiniteHits /> with "Show previous" button on first page 1`] = `
+<div
+  className="root"
+>
+  <button
+    className="undefined"
```

Comment:
We need to add the `loadPrevious` and `disabledLoadPrevious` CSS classes to the test.